recipes/ogdf/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ogdf" %}
+{% set version = "2020.02" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ogdf/ogdf/archive/catalpa-202002.tar.gz
+  sha256: 32f3f943675e55d1a23e62293d016dc1644b1176271d60f41788ade60c2d3fc5
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage('ogdf', max_pin='2020.02') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.1
+    - make  # [not win]
+    - ninja  # [win]
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libOGDF.so  # [linux]
+    - test -f $PREFIX/lib/libOGDF.dylib  # [osx]
+    - if not exist %LIBRARY_LIB%\\OGDF.lib exit 1  # [win]
+
+about:
+  home: http://www.ogdf.net/
+  license: GPL-2.0-only OR GPL-3.0-only
+  summary: OGDF is a self-contained C++ class library for the automatic layout of diagrams
+  
+  description: |
+    OGDF is a self-contained C++ class library for the automatic layout of diagrams.
+  doc_url: http://www.ogdf.net/doku.php
+  dev_url: https://github.com/ogdf/ogdf
+
+  license_file:
+    - LICENSE.txt
+    - LICENSE_GPL_v2.txt
+    - LICENSE_GPL_v3.txt
+    - LICENSE_abacus.txt
+    - LICENSE_backward.txt
+    - LICENSE_bandit.txt
+    - LICENSE_coin.txt
+    - LICENSE_minisat.txt
+    - LICENSE_pugixml.txt
+    - LICENSE_snowhouse.txt
+    - LICENSE_tinydir.txt
```

Comment:
Does this really include all of these other packages?

Reply:
It seems like `COIN_EXTERNAL_SOLVER_LIBRARIES ` can be used to point to the conda-forge version of coin: https://github.com/ogdf/ogdf/blob/master/doc/build.md  There are probably similar variables for the other packages

Reply:
> Does this really include all of these other packages?

It corresponds to all the licenses I have found in https://github.com/ogdf/ogdf/blob/master/LICENSE.txt. If there is more to include please let me know.

Reply:
> It seems like `COIN_EXTERNAL_SOLVER_LIBRARIES ` can be used to point to the conda-forge version of coin: https://github.com/ogdf/ogdf/blob/master/doc/build.md There are probably similar variables for the other packages

So you would like to package all those deps in conda forge or use the already packaged one and use it to build `ogdf`?

Reply:
> So you would like to package all those deps in conda forge or use the already packaged ones and use them to build ogdf?

Yep, exactly. This is for a couple of reasons, but the main ones are:

1. Consistent ABIs across all packages in an environment (eg only one copy of COIN) so that we know that the packages will work together
2. Use of shared libraries keep total size of all packages down because we aren't including a copy of COIN or zlib et cetra everytime another package uses it as a dependency
3. It pushes some of these license issues to the user's machine rather than having to worry about re-licensing here

Reply:
Generally, we want to avoid having conda packages vendor other package's SOs. This can cause environments that conflict so both packages would have the same SO's.

@wolfv has been working on the COIN build and so he should comment more.

Reply:
Ok, I totally understand the motivation here but this is a significant amount of work and I don't have the bandwidth for this at the moment. I am sorry for that because I know it took you time to review this PR. 

We can let this open and I'll try to get back to it at some point (probably not soon) and someone else can take the lead.

We can also merge as it is and let someone else tackle this in the feedstock directly.

Reply:
Yeah, I totally get that! It is a ton of work.  Thanks so much @hadim! Hopefully someone with more time can come along and pick this up